.github/workflows/cherryPick.yml

```diff
@@ -34,23 +34,20 @@ jobs:
     needs: [validateActor, createNewVersion]
     if: ${{ always() && fromJSON(needs.validateActor.outputs.IS_DEPLOYER) }}
     runs-on: ubuntu-latest
+    env:
```

Comment:
This is more concise syntax for what used to be the `Save correct NEW_VERSION to env` step